Created landing page

kbojarowska opened the pull request:
- Created landing page without pixelarts
- added pink square for better positioning (there would be pixelart located)
- created components (Text, Button)
- added mobile version for langing page
- added global colors file

Removed circle from the background because of problems with positioning

juliadlutek commented:
Przejrzałam kod i napisałam jedną uwagę od siebie. Poza tym wszystko wydaje się być w porządku, a strona prezentuje się bardzo ładnie. Super, że ogarnęłaś workflow i można podejrzeć efekt deployu. 👏

juliadlutek commented:
I reviewed the code and added one comment of my own. Besides that, everything seems to be fine, and the website looks very nice. Great job on handling the workflow, and it's awesome that we can preview the deployment results. 👏

juliadlutek commented:
I reviewed the code and added one comment. Besides that, everything seems to be fine, and the website looks very nice. Great job on handling the workflow, and it's awesome that we can preview the deployment results. 👏

juliadlutek commented:
One more question: Shouldn't we create a mobile version right away? I know we don't have a Figma design, but maybe everyone can customize their style as they see fit. It doesn't have to look perfect, I just suggest we make sure the elements don't overlap and change the size or flex direction if necessary for smaller screens. If we want to add a mobile version later in the project, it will require much more work. Let me know what you think about it.

kbojarowska commented:
> I reviewed the code and added one comment. Besides that, everything seems to be fine, and the website looks very nice. Great job on handling the workflow, and it's awesome that we can preview the deployment results. 👏

I cant see the comment you did :(

juliadlutek commented:
> > I reviewed the code and added one comment. Besides that, everything seems to be fine, and the website looks very nice. Great job on handling the workflow, and it's awesome that we can preview the deployment results. 👏
> 
> I cant see the comment you did :(

I'm not sure why you can't see it. Maybe I made some mistake, I don't have much experience in conducting code reviews on GitHub 🤔 
 
![image](https://github.com/kbojarowska/DoTheBeast/assets/75124863/7da00d4c-6457-444d-b141-ac4a19e563e2)

kbojarowska commented:
> > > I reviewed the code and added one comment. Besides that, everything seems to be fine, and the website looks very nice. Great job on handling the workflow, and it's awesome that we can preview the deployment results. 👏
> > 
> > 
> > I cant see the comment you did :(
> 
> I'm not sure why you can't see it. Maybe I made some mistake, I don't have much experience in conducting code reviews on GitHub 🤔
> 
> ![image](https://user-images.githubusercontent.com/75124863/282413853-7da00d4c-6457-444d-b141-ac4a19e563e2.png)

I added a global colors file and mobile version scss :)
